plugin/storage/es/options.go

```diff
@@ -240,6 +240,12 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 		"Use read and write aliases for indices. Use this option with Elasticsearch rollover "+
 			"API. It requires an external component to create aliases before startup and then performing its management. "+
 			"Note that "+nsConfig.namespace+suffixMaxSpanAge+" is not taken into the account and has to be substituted by external component managing read alias.")
+	flagSet.Bool(
+		nsConfig.namespace+suffixUseILM,
+		nsConfig.UseILM,
+		"(experimental) Option to enable ILM for jaeger span & service indices. Use this option with use-aliases"+
+			"It requires an external component to create aliases before startup and then performing its management."+
```

Comment:
Should we suggest instructions on how to do this via the `esRollover` script using the `init` mode?

Reply:
I think that should be added to documentation.